bridge/test/nftbridge/NFTBridge_test.js

```diff
@@ -153,18 +171,10 @@ contract("Bridge NFT", async function(accounts) {
         const tokenURI = "/ipfs/QmYBX4nZfrHMPFUD9CJcq82Pexp8bpgtf89QBwRNtDQihS";
         let totalSupply = 0; // is the amount of nft minted
 
-        let receipt = await this.token.safeMint(tokenOwner, tokenId, {
-          from: tokenOwner,
-        });
-        utils.checkRcpt(receipt);
+        await mintAndApprove(this.token, this.bridgeNft.address);
         totalSupply++;
 
-        receipt = await this.token.setTokenURI(tokenId, tokenURI);
-        utils.checkRcpt(receipt);
-
-        receipt = await this.token.approve(this.bridgeNft.address, tokenId, {
-          from: tokenOwner,
-        });
+        let receipt = await this.token.setTokenURI(tokenId, tokenURI);

```

Comment:
Please create a test with a token that does not have a custom Uri. it will help in the future

Reply:
I will do

Reply:
done at https://github.com/rsksmart/tokenbridge/pull/248/commits/5c5b7b5ec54603781de509724615d529d8089eb7 and https://github.com/rsksmart/tokenbridge/pull/248/commits/c52171392552917cb83ee86fb625e2b7d8c7a4d1